advocacy_docs/edb-postgres-ai/console/estate/agent/beacon-agent-ref.mdx

```diff
@@ -158,8 +158,7 @@ agent:
 provider:
   onprem:
     databases:
-      sales_reporting:
-        dsn: "$DSN"
+      - dsn: "$DSN"
```

Comment:
In https://github.com/EnterpriseDB/docs/pull/6105/files a resource_id key is used; are we skipping it here for compactness?

Reply:
Well, yes but it also conflicts with your edits which is what I want to resolve. Given the edits in here, I'd like to promote this PR and ensure nothing is lost from the other PR before we close it.